documentation/bin/createDocu.groovy

```diff
@@ -724,7 +754,8 @@ stages.each {key, value ->
 
     //remove details from parameter map
     stageStepKeys.each {stepKey ->
-        stageDescriptors."${key}".parameters.remove(stepKey)
+        if (stageDescriptors."${key}"?.parameters)
```

Comment:
Just for clarification: most parts of this PR deals with restoring `prepareDefaultValues`. This here seems to be unrelated.